conf/messages.en

```diff
@@ -322,21 +322,23 @@ supplementary.carrier.address.info = Carrier’s name and business address
 supplementary.carrier-details.help-item1 = Carrier’s EORI number, go to data element 3/32 Carrier Identification Number
 supplementary.carrier-details.help-item2 = Carrier’s name and business address, go to data element 3/31 Carrier Name and Address
 
-supplementary.representative.header = Who is the representative?
+
+supplementary.representative.title = Representation information
+supplementary.representative.legend = What type of representation was agreed?
 supplementary.representative.hint = Enter the representative’s EORI number or name and business address. If no representative was used, leave this page blank and continue.
-supplementary.representative.eori.info = Representative’s EORI number
+supplementary.representative.eori.legend = What is the representative’s EORI number?
+supplementary.representative.eori.hint = If you are an agent representing another agent, tell us their EORI number, otherwise leave it blank
```

Comment:
In general we should use `declaration` instead of `supplementary` in the `messages.en`. This is the old approach when we build the screen just for supplementary dec. We have this in a lot of places, but we're trying to change it to correct one, when we're touching the message keys

Reply:
I agree, but here we have a singular case. Messages like that are shared across different pages. I decided to keep them like that to be consistent with what they are today and to avoid potential conflicts.

Reply:
Are you sure they are used elsewhere Maurício?   I think they are only used on representative_details screen (and probably the Form object).   So a rename should be fairly painless.